src/couch/src/couch_native_process.erl

```diff
@@ -65,10 +72,12 @@ prompt(Pid, Data) when is_list(Data) ->
 
 % gen_server callbacks
 init([]) ->
-    {ok, #evstate{ddocs=dict:new()}}.
+    V = config:get("query_server_config", "os_process_idle_limit", "300"),
+    Idle = list_to_integer(V) * 500,
```

Comment:
Ooops! Was debugging why timeouts weren't working and forgot to undo that debug change. squashvertipushing.